src/components/Logo/Logo.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Image from 'next/image';
+import logo from '../../../public/img/logo.png';
```

Comment:
The `png` image here looks so pixelated. Kindly download the SVG logo from the Figma design and use that one instead.